clients/vscode/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [0.0.56](https://github.com/prisma/vscode/compare/prisma-vscode@0.0.47...prisma-vscode@0.0.56) (2020-05-11)
+
+**Note:** Version bump only for package prisma-vscode
+
```

Comment:
Probably not intentional? By some script that should be disabled?

Reply:
no, I think that was added by publishing with lerna instead of vsce, I'll remove that